File: src/test/java/com/fifo/ticketing/TicketingApplicationTests.java
package com.fifo.ticketing;

import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.ActiveProfiles;
import org.springframework.test.context.TestPropertySource;
import org.springframework.test.context.jdbc.Sql;

@SpringBootTest
@ActiveProfiles("ci")
@TestPropertySource(locations = "classpath:application-ci.yml")
class TicketingApplicationTests {

    @Test
    void contextLoads() {
    }

}

File: src/test/java/com/fifo/ticketing/TicketingApplicationTests.kt
package com.fifo.ticketing

import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.context.ActiveProfiles
import org.springframework.test.context.TestPropertySource

@SpringBootTest
@ActiveProfiles("ci")
@TestPropertySource(locations = ["classpath:application-ci.yml"])
class TicketingApplicationTests {

    @Test
    fun contextLoads() {
    }
}